src/status_im/chat/models/input.cljs

```diff
@@ -25,9 +25,9 @@
 (defn text-ends-with-space?
   "Returns true if the last symbol of `text` is space"
   [text]
```

Comment:
Do we even need to implement this function by ourself ? There is [`clojure.string/ends-with?`](https://github.com/clojure/clojurescript/blob/master/src/main/cljs/clojure/string.cljs#L255) function which we can use this way:
```clj
(def text-ends-with-space? #(str/ends-with? % const/spacing-char))
```
Which is so super simple, that maybe even unit testing it is not necessary. By the way const `spacing-char` has very misleading name, as in Clojure (not in ClojureScript where characters are just one item strings), characters are treated differently from strings and it may be good to stick to some common standard :)